app/src/main/java/com/concordia/mcga/helperClasses/DatabaseHelper.java

```diff
@@ -5,22 +5,21 @@
 import android.database.sqlite.SQLiteException;
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
-
+import com.concordia.mcga.exceptions.MCGADatabaseException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.sql.SQLException;
 
 
 //Part of the code was taken from https://blog.reigndesign.com/blog/using-your-own-sqlite-database-in-android-applications/
 public class DatabaseHelper extends SQLiteOpenHelper {
+
     private static final String DATABASE_NAME = "development.db";
-    private static SQLiteDatabase db;
 
     //The Android's default system path of our application database.
     private static String databasePath;
-    private static Context context;
+    private Context context;
     private static DatabaseHelper databaseHelper;
```

Comment:
Do not place Android context classes in static fields (static reference to DatabaseHelper which has field context pointing to Contex); this is a memory leak (and also breaks Instant Run)

http://stackoverflow.com/questions/37709918/warning-do-not-place-android-context-classes-in-static-fields-this-is-a-memory